server/database/config/dbbuild.js

```diff
@@ -0,0 +1,7 @@
+const { join } = require('path');
+const { readFileSync } = require('fs');
+const dbconnection = require('./dbconnection');
+
+let sql = readFileSync(join(__dirname, 'dbbuild.sql')).toString();
+if (process.env.NODE_ENV === 'test') { sql += readFileSync(join(__dirname, 'insertFakeData')).toString(); }
```

Comment:
I suggest adding the SQL line in a new line, it would be readable and you are using the curly braces anyway so why not